resource/locators.py

```diff
@@ -0,0 +1,39 @@
+from selenium.webdriver.common.by import By
+
+
+class LoginPageLocators():
+    LOGIN_FORM = (By.CSS_SELECTOR, "#login_form")
```

Comment:
I suggest to incapsulate locators inside corresponding class LoginPage, to not waste the time listing project and jumping from page class to locators class